featuretools/tests/primitive_tests/test_rolling_primitive_utils.py

```diff
@@ -475,4 +585,78 @@ def test_roll_series_with_non_offset_string_inputs(rolling_series_pd):
         "Please either change gap to be numeric or change window length to be an offset alias."
     )
     with pytest.raises(TypeError, match=error):
-        roll_series_with_gap(rolling_series_pd, window_size=7, gap="2d").max()
+        roll_series_with_gap(
+            rolling_series_pd,
+            window_size=7,
+            gap="2d",
+            min_periods=1,
+        ).max()
+
+
+@pytest.mark.parametrize(
+    "primitive",
+    [RollingCount, RollingMax, RollingMin, RollingMean, RollingSTD, RollingTrend],
+)
+@patch("featuretools.primitives.rolling_primitive_utils.apply_roll_with_offset_gap")
+def test_no_call_to_apply_roll_with_offset_gap_with_numeric(
```

Comment:
this test was moved to the utils test since it is testing the utils function more than the actual primitives